components/proxy/src/main/kotlin/com/hotels/styx/ObjectTags.kt

```diff
@@ -0,0 +1,18 @@
+/*
+  Copyright (C) 2013-2019 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
+package com.hotels.styx
+
+fun lbGroupTag(name: String) = "lbGroup=$name"
```

Comment:
I guess you don't need it now (YAGNI, etc.) but do you also see adding functions here to extract the RHS from a tag ? e.g. `fun lbGroupTag(tags: Set<String>) : String`

Reply:
Hi. Yes. Right now it is unnecessary. But will add if or when necessary.